kairon/shared/pyscript/shared_pyscript_utils.py

```
@@ -120,6 +120,35 @@ def delete_data(collection_id: str, user: Text, bot: Text = None):
+        from kairon.shared.data.customer_order_processor import CustomerOrderProcessor
+        return CustomerOrderProcessor.get_order(bot=bot, order_id=order_id)
+
+    @staticmethod
+    def get_customer_details(sender_id: str, bot: Text = None) -> dict:
+        if not bot:
+            raise Exception("Missing bot id")
+        from kairon.shared.data.customer_order_processor import CustomerOrderProcessor
+        return CustomerOrderProcessor.get_customer(bot=bot, sender_id=sender_id)
+
+    @staticmethod
+    def upsert_customer(sender_id: str, persona_type: str = None, payload: dict = None, bot: Text = None) -> dict:
```

Ruff (0.16.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)